app/components/component-docs/dataDisplay/CoreChip.docs.js

```diff
@@ -310,48 +343,42 @@ Use the avatar prop to add an avatar or use the icon prop to add an icon."
 </CoreBox>
         `}
         renderElement={
-          <CoreBox /*sx={{ width: 100 }}*/>
-            <CoreChip
-              // sx={{
-              //   "& .MuiChip-label": {
-              //     display   : "block",
-              //     whiteSpace: "normal",
-              //   },
-              //   height: "auto",
-              // }}
-              label="This is a chip that has multiple lines."
-            />
-          </CoreBox>
+          <>
+
+            <CoreBox /*sx={{ width: 100 }}*/>
+              <CoreChip
+                // sx={{
+                //   height: 'auto',
+                //   '& .MuiChip-label': {
+                //     display: 'block',
+                //     whiteSpace: 'normal',
+                //   },
+                // }}
+                label="This is a chip that has multiple lines."
+              />
+            </CoreBox>
+          </>
         }
       />
 
       <CodeSample
         title="Chip array"
         description="An example of rendering multiple chips from an array of values. Deleting a chip removes it from the array. Note that since no onClick prop is defined, the Chip can be focused, but does not gain depth while clicked or touched."
-        code={`
-<CorePaper
-  // sx={{
-  //   display: 'flex',
-  //   justifyContent: 'center',
-  //   flexWrap: 'wrap',
-  //   listStyle: 'none',
-  //   p: 0.5,
-  //   m: 0,
-  // }}
+        code={`<CorePaper styleClasses={[CoreClasses.DISPLAY.FLEX, CoreClasses.ALIGNMENT.JUSTIFY_CONTENT_CENTER, CoreClasses.PADDING.P1]}
   component="ul"
 >
   {[
-    { key: 0, label: 'Angular' },
-    { key: 1, label: 'jQuery' },
-    { key: 2, label: 'Polymer' },
-    { key: 3, label: 'React' },
-    { key: 4, label: 'Vue.js' },
+    { key: 0, label: "Angular" },
```

Comment:
Change it to Wrappid packages name or something relatable to Wrappid

Reply:
Good thought